job-dsls/jobs/tools/tools_provisioning_verdaccio.groovy

```diff
@@ -13,7 +13,7 @@ def final JOB_NAMES = [
         "provision-verdaccio-service"  : [
                 params : [
                         [name: "IMAGE", default: "rhel-7.6-server-x86_64-released", description: "The name of the image to be used for machine creation." ],
```

Comment:
Please change the rhel-7.6 image here as well, thanks!

Reply:
DONE